.gitignore

```diff
@@ -1,3 +1,4 @@
+.gitignore
```

Comment:
I'd expect to commit the .gitignore file?

Reply:
I didn't mean to edit your gitignore, I added some additional files to it gitignore and put gitignore to itself first so it wouldn't track this.  However, it seems to have added the gitignore change. Feel free to undo this.